### ISSUE TITLE
[BUG] `addImage()` with uppercase `path` prop causes "needs to repair presentation" in Windows and MacOS

### ISSUE THREAD

**Issue by mamodo123** [gitbrent/PptxGenJS#860]
Version: 3.3.1 from CDN

Hello. I'm trying to create a presentation only with images. When I open it in linux, it's ok, but in windows and macOs the presentation needs to be repaired. all images are added by they base64 url, like this:

`slide.addImage({
                path: item.toDataURL(),
                x: xPosition + padding,
                y: (defaultSize[1] - ppHeight) / 2,
                w: ppWidth,
                h: ppHeight
            });`

the broken image is always one of two images in a single page.

![image](https://user-images.githubusercontent.com/47646223/97989297-18fa1880-1dbd-11eb-8c9e-25e015fbe58a.png)

**Comment by gitbrent** [comment]
There's not enough information provided to diagnose this.

The demo produces quite a lot of images from various sources. Try the image demo to ensure there are no issues, then check your image source.

### [Image Demo](https://gitbrent.github.io/PptxGenJS/demo/#images)

**Comment by aviralgoyal** [comment]
I had the same issue, in order to resolve this please ensure that all images been added have the extension in lower case and not uppercase.

**Comment by WilliamWaterworth** [comment]
No issues with Libre Office or Slack. Any capital character in the filename breaks Powerpoint, so I used a workaround:
```
fetch("/path/to/FILE.svg")
	.then(r => r.text())
	.then(s => "data:image/svg+xml;base64," + window.btoa(s))
```
Pass in data instead of source to addImage, and obviously change the image type as needed.

**Comment by gunner13** [comment]
> 

You saved the day.

**Comment by gunner13** [comment]
> I had the same issue, in order to resolve this please ensure that all images been added have the extension in lower case and not uppercase.


You saved the day.

**Comment by gitbrent** [comment]
Oh, that's the information I needed.

FAIL Test Case:
```javascript
let pptx = new PptxGenJS();
let slide = pptx.addSlide();

slide.addImage({
  x: 1, y: 1, w: 4, h: 3,
  path: "https://raw.githubusercontent.com/gitbrent/PptxGenJS/master/demos/common/images/UPPERCASE.PNG"
});

pptx.writeFile({ fileName: "PptxGenJS-issue860.pptx" });
```

**Comment by gitbrent** [comment]
This is issue is fixed in release v3.11.0

![Screen Shot 2022-07-24 at 15 08 33](https://user-images.githubusercontent.com/7218970/180664047-7f6240b9-f416-4ae5-ba93-adf60caa6dae.png)
